### PR TITLE
[WIP] [Propose] pg>: Output sql query log

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
Hello 

I would like to know what SQL query executes in digdag run operation.

What do you think this change?

### Current output

```
2017-08-23 20:19:09 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=0=1): pg>: select.sql
2017-08-23 20:19:10 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=1=2): pg>: select.sql
2017-08-23 20:19:10 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=2=3): pg>: select.sql
```

### Proposed change

```
2017-08-23 20:19:44 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=0=1): pg>: select.sql
2017-08-23 20:19:44 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=0=1): select *
2017-08-23 20:19:44 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=0=1):  from embulk_test
2017-08-23 20:19:44 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=0=1):  where id = 1
2017-08-23 20:19:44 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=1=2): pg>: select.sql
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=1=2): select *
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=1=2):  from embulk_test
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=1=2):  where id = 2
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=2=3): pg>: select.sql
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=2=3): select *
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=2=3):  from embulk_test
2017-08-23 20:19:45 +0900 [INFO] (0017@[0:default]+test+task^sub+for-0=id=2=3):  where id = 3
```

### CODE REVIEW

**Comment by komamitsu** [comment]
It's an interesting feature!

I think it would be great if this feature has these improvements.

1. SQL statement can be really huge and/or contain security sensitive values. So, this feature should be enabled/disabled by configuration (default: disabled)

2. It looks the current change isn't covering whole SQL statements (e.g. COMMIT). So, it needs more work

3. This feature should be available over other operators related to `AbstractJdbcConnection` (e.g. `redshift` operator)

BTW, if this feature isn't so urgent, we can take care of it later.

File: digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java [hunk 1]
```diff
@@ -79,6 +85,14 @@ public TransactionHelper getStrictTransactionHelper(String statusTableSchema, St
         return new PgPersistentTransactionHelper(statusTableSchema, statusTableName, cleanupDuration);
     }
 
+    private void loggingExecuteSQL(String sql)
+    {
+        for(String line: sql.split("\r?\n"))
```
Comment (komamitsu) [inline digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java]: [code style]
This line should be
```
        for (String line: sql.split("\r?\n")) {
```

**Comment by yoyama** [comment]
@hiroyuki-sato How about this PR?  I think this is good one.  
Anyway I postpone this  to 0.9.35 or later.

**Comment by hiroyuki-sato** [comment]
@yoyama Thank you for the comment. This PR needs to improve codes. I'll take a look.

**Comment by hiroyuki-sato** [comment]
I'll develop this feature on #1037.

**Pull request closed without merging**